### PR TITLE
fix bug on nav bar buttons

### PR DESCRIPTION
**Pull request by AlexPD93**
Now only the shopping cart button in the nav bar will have a number next to it if there are items in the cart #14

### CODE REVIEW

**Review by halimahexe — APPROVED** [review]
LGTM - I like how you used a boolean to manage this!

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -5,15 +5,26 @@ type buttonProps = {
...
+  icon,
+  size,
+  cartButton,
+}: buttonProps) {
   const { cartQuantity } = useCart();
 
   return (
     <button onClick={buttonClick} className={size}>
-      <span>{cartQuantity === 0 ? "" : cartQuantity} </span>
+      {cartButton ? (
```
Comment (diddydeeable) [inline src/components/Button/Button.tsx:23]: if you wanted to shorten it, you could use the same syntax you used below for icon

`			{cartButton && <span>{cartQuantity === 0 ? '' : cartQuantity} </span>}`

File: src/components/NavBar/NavBar.tsx
Comment (diddydeeable) [inline src/components/NavBar/NavBar.tsx:1]: useNavigate is really cool

**Review by diddydeeable — APPROVED** [review]
really cool button @AlexPD93 - love that the number changes dynamically!!

**Pull request merged**